spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfigurationTests.java

```diff
@@ -88,7 +89,8 @@ public void setup() {
 				DataFlowServerConfiguration.class, PropertyPlaceholderAutoConfiguration.class,
 				RestTemplateAutoConfiguration.class, HibernateJpaAutoConfiguration.class, WebConfiguration.class,
 				SchedulerConfiguration.class, JacksonAutoConfiguration.class, SimpleTaskAutoConfiguration.class,
-				ResourceLoadingAutoConfiguration.class, ComposedTaskRunnerConfigurationProperties.class);
+				ResourceLoadingAutoConfiguration.class, ComposedTaskRunnerConfigurationProperties.class,
+				H2ServerConfiguration.class);
```

Comment:
I don't think this is needed since DataFlowServerConfiguration brings it in. 

**[ALREADY-IN-CODE]** Neither does the WebConfiguration.

Reply:
I removed the WebConfiguration and H2ServerConfiguration